save-backend/src/main/kotlin/com/saveourtool/save/backend/service/LnkUserProjectService.kt

```diff
@@ -117,7 +118,7 @@ class LnkUserProjectService(
      * @param userId
      * @return list of [Project]s that are connected to user with [userId]
      */
-    fun getAllProjectsByUserId(userId: Long): List<Project> = lnkUserProjectRepository.findByUserId(userId)
+    fun getCreatedProjectsByUserId(userId: Long): List<Project> = lnkUserProjectRepository.findByUserIdAndProjectStatus(userId, ProjectStatus.CREATED)
```

Comment:
Maybe `getExistingProjectsByUserId` or `getNonDeletedProjectsByUserId`? Because it kinda sounds like 'get projects created by this user'

Reply:
Right, will fix it